apps/engmanagement/src/pages/api/skill/[...skillRecordings].ts

```diff
@@ -5,7 +5,7 @@ import SkillRecordings, {
 import prisma from '../../../db'
 
 export const skillOptions: SkillRecordingsOptions = {
-  prismaClient: prisma,
+  prismaClient: prisma as any,
```

Comment:
not super stoked on the any use, but it works and on the `skill-api` side we expect the client to match and it does

it'd be nice to have the client generated centrally, but I think it is built around a specific `DATABASE_URL`. Need more research there.